plugins/outputs/kinesis/kinesis.go

```diff
@@ -54,6 +56,11 @@ var sampleConfig = `
   streamname = "StreamName"
   ## PartitionKey as used for sharding data.
   partitionkey = "PartitionKey"
+  ## If set the paritionKey will be a random UUID on every put.
+  ## This allows for scaling across multiple shards in a stream.
+  ## This will cause issues with ordering.
+  randomize_partitionKey = false
```

Comment:
Rename to `use_random_partitionkey`.  Mention that if this is set then `partitionkey` is ignored.  Can you explain a little about the ordering issues (perhaps in the README)